app/views/provider/admin/messages/outbox/new.html.erb

```diff
@@ -19,8 +19,9 @@
       <% end %>
     </li>
 
-    <%= form.input :subject, :as => :string, :label => 'Subject:' %>
-    <%= form.input :body, :input_html => {:rows => 10}, :label => 'Body:' %>
+    <%= form.input :subject, :as => :string, :label => 'Subject:', input_html: { required: true } %>
+    <%= form.input :body, :input_html => {:rows => 10, required: true}, :label => 'Body:' %>
+
```

Comment:
I see this also being done this way in app/views/buyers/accounts/bulk/send_emails/new.html.erb, app/views/buyers/applications/bulk/send_emails/new.html.erb, app/views/buyers/service_contracts/bulk/send_emails/new.html.erb, but if you look at app/views/buyers/users/edit.html.erb line 13, you will see you can do `required: true` like this:

```
<%= form.input :password, required: true, input_html: { autocomplete: 'off' } %>
```

This also looks like the preferred way as per the docs also:
https://github.com/formtastic/formtastic

Reply:
yes both are correct ways , but if input_html is already used there so we can keep required: true inside it also.
I have seen it in my earlier experiences that when we use it inside it input_html it works for all browsers.

Reply:
@jkeam @josemigallas if all looks good to you guys now can we merge this?